test/carelink/testCommon.js

```diff
@@ -79,4 +80,35 @@ describe('common', function() {
       expect(common.isSuspectedNewDevice(notEmptyObj)).to.be.false;
     });
   });
+
+  describe('isMgDL', function() {
+    it('should be a function', function() {
+      expect(common.isMgDL).to.exist;
+      expect(typeof common.isMgDL).to.equal('function');
+    });
+
+    it('should return true on `mg/dL`', function() {
+      expect(common.isMgDL('mg/dL')).to.be.true;
+    });
+
+    it('should return false on `mmol/L`', function() {
+      expect(common.isMgDL('mmol/L')).to.be.false;
+    });
+
+    it('should throw an error on `foo`', function() {
+      var fn = function() { common.isMgDL('foo'); };
+      expect(fn).to.throw(Error);
+    });
+  });
+
+  describe('convertBackToMmol', function() {
+    it('should be a function', function() {
+      expect(common.convertBackToMmol).to.exist;
+      expect(typeof common.convertBackToMmol).to.equal('function');
+    });
+
+    it('should return a floating-point number with one significant digit', function() {
+      expect(String(common.convertBackToMmol(99.08))).to.equal('5.5');
```

Comment:
Yup, this is an example inspired by real life. They are clearly converting *to* mg/dL in the `Raw-Values`. For some data types we do still get the actual mmol/L value in a separate column, but for the types that don't have a separate column (in the settings), we only see the conversion.